uber/templates/preregistration/preregbase.html

```diff
@@ -18,7 +15,7 @@
     };
     setInterval(checkCap, 1000 * 1 * 90);
 
-{% if COLLECT_FULL_ADDRESS %}
+{% if c.COLLECT_FULL_ADDRESS %}
```

Comment:
Yeah, I kept it in as a config option because I know Dom REALLY wants to be able to flip a switch to start collecting addresses for MagClassic.  I'm pretty ambivalent about tackling that this year, but since this if statement was already in place I figured I'd leave it there.